src/components/Schedule/ProgressBar.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react";
+
+interface IProps {
+  currentReservation: number;
+  maxReservation: number;
+}
+
+export const ProgressBar: FC<IProps> = ({
+  currentReservation,
+  maxReservation
+}) => (
+  <div>
+    <div className="flex justify-between pt-6">
```

Comment:
add the classnames `text-sm font-lato font-bold` to this div, then you won't need to repeat it a few times in the children divs